components/docs-chef-io/content/automate/ha_aws_deploy_steps.md

```diff
@@ -50,6 +54,15 @@ Follow the steps below to deploy Chef Automate High Availability (HA) on AWS (Am
 
 Run the following steps on Bastion Host Machine:
 
+{{< note >}}
+
+- Make sure that bastion machine is in the same vpc, as mention in `config.toml`, otherwise we need to do [vpc peering](https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html).
+- Use subnet-id instead of CIDR block in `config.toml`, to avoid the subnet conflict. If we use CIDR block, will fail if an consecutive cidr block are not available.
+- If you choose `backup_config` as `s3` then provide the bucket name to field `s3_bucketName`. If `s3_bucketName` exist it is directly use for backup configuration and if it doesn't exist then deployment code will create `s3_bucketName`.
```

Comment:
instead of "then deployment code will create" how about "then deployment process will create"